src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,16 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1024px;
+  margin: 0 auto;
```

Comment:
8. [BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
agree

Reply:
let's fix it🙂